src/components/ComparisonMap/ComparisonMap.vue

```diff
@@ -54,8 +63,12 @@
       },
     },
 
-    async mounted () {
-      await this.initializeMap()
+    watch: {
+      async isLoading (newValue) {
+        if (!newValue) {
```

Comment:
Can you document this function? 
- When is newValue falsey?
- Why should we then initialize the map? 
- Why is this async?